src/openforms/js/components/admin/form_design/MissingComponentTranslationsWarning.js

```diff
@@ -0,0 +1,141 @@
+import FormioUtils from 'formiojs/utils';
+import PropTypes from 'prop-types';
+import React, {useState} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import MessageList from 'components/admin/MessageList';
+import Modal from 'components/admin/Modal';
+import {ChangelistColumn, ChangelistTable} from 'components/admin/tables';
+import {
+  extractComponentLiterals,
+  getSupportedLanguages,
+} from 'components/formio_builder/translation';
+
+const LANGUAGES = getSupportedLanguages();
```

Comment:
this used to be inside the react component (via jsonScriptToVar) which does DOM operations, and you absolute do not want to do that in React. DOM interaction is comparatively expensive, and having every render to DOM operations removes the advantages from using React.

This is also information that doesn't change once the page is loaded, so evaluating it once at the module level is appropriate.